DataProducts/inc/STMTestBeamEventInfo.hh

```diff
@@ -0,0 +1,22 @@
+#ifndef DataProducts_STMTestBeamEventInfo_hh
+#define DataProducts_STMTestBeamEventInfo_hh
+
+namespace mu2e {
+  class STMTestBeamEventInfo {
+  public:
+    enum TrigType { unknown=-1, internal=0, external=1 };
+    STMTestBeamEventInfo() : _trigType(TrigType::unknown), _triggerTime(0) { }
+
+    void trigType(uint16_t type) { _trigType = TrigType(type);}
+    void triggerTime(uint64_t time) { _triggerTime = time;}
+
+    TrigType trigType() const { return _trigType;}
+    uint64_t triggerTime() const { return _triggerTime;}
+
+  private:
+    TrigType _trigType;
```

Comment:
A style preference you might consider.  You can write these as:

TrigType _trigType = TrigType::unknown;  // or maybe just unknown ?
uint64_t _triggerTime = 0;

Then they are automatically set in every c'tor and you can get rid of the initializer list.

Reply:
Done in f78aa57